lib/msal-angular/src/index.ts

```diff
@@ -1,6 +0,0 @@
-export { BroadcastService } from './broadcast.service';
```

Comment:
Should we take a look at what we export from index.ts in all libs?

Reply:
I just renamed this file to `public_api.ts` (which is an Angular pattern), and added the exported constants.

Is there something specific you are curious about?

Reply:
We can talk offline.